### PR TITLE
Primary and non-primary branch content

### PR DESCRIPTION
**Pull request by danieltprice**
- Primary branch:
https://neon-next-git-dprice-branch-types-neondatabase.vercel.app/docs/manage/branches#primary-branch
-Non-primary branches:
https://neon-next-git-dprice-branch-types-neondatabase.vercel.app/docs/manage/branches#non-primary-branch
- Glossary entries:
https://neon-next-git-dprice-branch-types-neondatabase.vercel.app/docs/reference/glossary#primary-branch
https://neon-next-git-dprice-branch-types-neondatabase.vercel.app/docs/reference/glossary#non-primary-branch
- Billing page. Mention the 20 simultaneously active compute limit (under **Notes** after the table)
https://neon-next-git-dprice-branch-types-neondatabase.vercel.app/docs/introduction/billing#neon-plans
- Refreshed branch screen captures
- Minor edits and revisions

### CODE REVIEW

File: content/docs/introduction/billing.md [hunk 1]
```diff
@@ -37,8 +37,9 @@ See [Billing and usage metrics explained](#billing-and-usage-metrics-explained)
 **Notes:**
 
 - The limits described above are plan defaults. If you want to adjust the limits to tailor a plan to your specific requirements, please contact [Sales](https://neon.tech/contact-sales).
-- The Pro plan has a 200 GB safety limit on storage to protect your account from unintended usage but supports any data size. To increase the safety limit for your Pro account, simply submit a [Sales](https://neon.tech/contact-sales) request with the data storage size that you require.
-- The Free Tier has a 100 hour _compute active time per month_ limit, but you are always able to connect to the compute endpoint assigned to the primary branch of your Neon project, which ensures that access to data on the primary branch of your project is never interrupted.
+- The Pro plan has a 200 GB safety limit on storage to protect your account from misuse but supports any data size. To increase the safety limit for your Pro account, simply submit a [Sales](https://neon.tech/contact-sales) request with the data storage size that you require.
```
Comment (ololobus) [inline content/docs/introduction/billing.md:40]: 'Safety' is internal reasoning, I'd prefer to don't reference it thins way in the docs. And rather call 'default' with the same suggestion about increasing it via support request

File: content/docs/introduction/billing.md [hunk 2]
```diff
@@ -37,8 +37,9 @@ See [Billing and usage metrics explained](#billing-and-usage-metrics-explained)
...
+- The Pro plan has a safety limit of 20 simultaneously active computes to protect your account from misuse but supports a higher limit if required.
```
Comment (ololobus) [inline content/docs/introduction/billing.md:41]: Same here

Comment (ololobus) [reply content/docs/introduction/billing.md:41]: Let's also mention here that primary branch isn't affected by the active endpoints limit?

I see it's mentioned on the different page, but for free tier we note this on this page too, so maybe better to be consistent

**Review by ololobus — APPROVED** [review]
Left some comments, but LGTM in overall

File: content/docs/introduction/billing.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/introduction/billing.md:40]: Replaced with "default"

**Pull request merged**